doc/developer/Building_FRR_on_CentOS6.rst

```diff
@@ -80,6 +94,24 @@ Please note that ``CentOS 6`` needs to keep python pointing to version
 2.6 for ``yum`` to keep working, so don't create a symlink for python2.7
 to python
 
+Install newer ``Sphinx-Build`` based on ``Python 2.7``
+
+::
+    curl -O http://springdale.math.ias.edu/data/puias/computational/6/x86_64/python27-babel-0.9.6-2.sdl6.1.noarch.rpm
+    curl -O http://springdale.math.ias.edu/data/puias/computational/6/x86_64/python27-jinja2-2.7.2-2.sdl6.noarch.rpm
+    curl -O http://springdale.math.ias.edu/data/puias/computational/6/x86_64/python27-markupsafe-0.11-4.puias6.x86_64.rpm
+    curl -O http://springdale.math.ias.edu/data/puias/computational/6/x86_64/python27-pygments-1.4-4.sdl6.1.noarch.rpm
+    curl -O http://springdale.math.ias.edu/data/puias/computational/6/x86_64/python27-docutils-0.8.1-3.sdl6.2.noarch.rpm
+    curl -O http://springdale.math.ias.edu/data/puias/computational/6/x86_64/python27-imaging-1.1.7-6.puias6.x86_64.rpm
+    curl -O http://springdale.math.ias.edu/data/puias/computational/6/x86_64/python27-sphinx-1.1.3-1.sdl6.1.noarch.rpm 
+    sudo yum install ./python27-markupsafe-0.11-4.puias6.x86_64.rpm \
+      ./python27-babel-0.9.6-2.sdl6.1.noarch.rpm \
+      ./python27-jinja2-2.7.2-2.sdl6.noarch.rpm \
+      ./python27-pygments-1.4-4.sdl6.1.noarch.rpm \
+      ./python27-docutils-0.8.1-3.sdl6.2.noarch.rpm \
+      ./python27-imaging-1.1.7-6.puias6.x86_64.rpm \
+      ./python27-sphinx-1.1.3-1.sdl6.1.noarch.rpm 
+
```

Comment:
I'm not opposed to this, but I've tested python-sphinx10 and it works fine with the exception that sphinx-1.0-build is used,

Reply:
rather than use explicit curl, should add repo, e.g.:
 cat > /etc/yum.repos.d/sphinx-build.repo
### Name: RPM Repository for RHEL 6 - Sphinx-Build
### URL: http://springdale.math.ias.edu/data/puias/computational
[puias-computational]
name = RPM Repository for RHEL 6 - Sphinx-Build
baseurl = http://springdale.math.ias.edu/data/puias/computational/$releasever/$basearch
#mirrorlist = 
#mirrorlist = 
enabled = 1
protect = 0
gpgkey = 
gpgcheck = 0